models/ssd_detector/README.md

```diff
@@ -62,10 +62,18 @@ models/ssd_detector directory. You'll see the `ssd_detector` folder with sample
 demonstrating how to train a MobileNetV2-based SSD object detector.
 
 We provide 2 predifined configuration:
-* Vehicles and license plates detector [models/ssd_detector/vlp/config.py](vlp/config.py).
-* COCO object detector detector [models/ssd_detector/coco/config.py](coco/config.py) on COCO dataset [data/coco/README.md](../../data/coco/README.md).
+* Vehicles and license plates detector.
+  ![VLP detection](vlp/docs/sample.jpg "Example of VLP detector inference")
+
+  - Configuration file: [models/ssd_detector/vlp/config.py](vlp/config.py).
+  - Trained model: [MobileNet v2 0.35 256x256](https://download.01.org/openvinotoolkit/training_toolbox_tensorflow/models/ssd_detector/vlp/0123_ssd_mobilenet_v2_0.35_barrier_256x256.zip).
```

Comment:
Seems like this archive is broken.

Reply:
I think the problem is connected to hosting. I've tried to download the archive multiple times, but it was downloaded only once.